### PR TITLE
util/admin: clean up and add comparing table and index count func

### PR DESCRIPTION
**Pull request by zimulala**
Remove 'GetTableRecordsCount' and 'GetIndexRecordsCount'. Because its result is the same as count (*), and it's slower.
Add 'CompareIndexCount'. I will use it in next PR.

### CODE REVIEW

File: util/admin/admin_test.go [hunk 1]
```diff
@@ -466,6 +418,8 @@ func (s *testSuite) testIndex(c *C, tb table.Table, idx table.Index) {
 	// table     data (handle, data): (1, 10), (2, 20), (3, 30)
 	key = tablecodec.EncodeRowKey(tb.Meta().ID, codec.EncodeInt(nil, 4))
 	txn.Delete(key)
+	key = tablecodec.EncodeRowKey(tb.Meta().ID, codec.EncodeInt(nil, 5))
```
Comment (zimulala) [inline util/admin/admin_test.go]: Keep it consistent with the comments.
And I want table count is different with index count.

**Comment by zimulala** [comment]
PTAL @shenli @coocood

File: util/admin/admin.go [hunk 2]
```diff
@@ -167,27 +171,38 @@ type RecordData struct {
...
-func GetIndexRecordsCount(sc *stmtctx.StatementContext, txn kv.Transaction, kvIndex table.Index, startVals []types.Datum) (int64, error) {
-	it, _, err := kvIndex.Seek(sc, txn, startVals)
+func getCount(ctx context.Context, sql string) (int64, error) {
+	rows, _, err := ctx.(sqlexec.RestrictedSQLExecutor).ExecRestrictedSQL(ctx, sql)
 	if err != nil {
 		return 0, errors.Trace(err)
 	}
-	defer it.Close()
+	if len(rows) != 1 {
+		return 0, errors.New("can not get count")
```
Comment (shenli) [inline util/admin/admin.go]: This should not happen, we need to log the sql and len(rows) here.

File: util/admin/admin.go [hunk 3]
```diff
@@ -167,27 +171,38 @@ type RecordData struct {
...
-	for {
-		_, _, err := it.Next()
-		if terror.ErrorEqual(err, io.EOF) {
-			break
-		} else if err != nil {
-			return 0, errors.Trace(err)
+// CompareIndexCount compares indexes count with table count.
+// It returns nil if the count from the index is equal to the count from the table columns,
+// otherwise it returns an error with a different information.
+func CompareIndexCount(ctx context.Context, dbName, tableName string, indexes []string) error {
```
Comment (shenli) [inline util/admin/admin.go]: CheckIndicesCount

**Comment by zimulala** [comment]
PTAL @coocood

**Comment by zimulala** [comment]
PTAL @shenli

**Comment by shenli** [comment]
LGTM

File: util/admin/admin.go [hunk 4]
```diff
@@ -167,27 +171,38 @@ type RecordData struct {
...
-		_, _, err := it.Next()
-		if terror.ErrorEqual(err, io.EOF) {
-			break
-		} else if err != nil {
-			return 0, errors.Trace(err)
+// CheckIndicesCount compares indices count with table count.
+// It returns nil if the count from the index is equal to the count from the table columns,
+// otherwise it returns an error with a different information.
+func CheckIndicesCount(ctx context.Context, dbName, tableName string, indices []string) error {
+	sql := fmt.Sprintf(`SELECT COUNT(*) FROM %s.%s"`, dbName, tableName)
```
Comment (lamxTyler) [inline util/admin/admin.go]: Do we need to surround the name with backtick to avoid names like `a b`?

**Comment by zimulala** [comment]
PTAL @lamxTyler

**Pull request merged**